tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -153,15 +153,24 @@ public SubChannel makeSubChannel(String service) {
         return this.makeSubChannel(service, Connection.Direction.NONE);
     }
 
-    public void shutdown(boolean sync) throws InterruptedException, ExecutionException {
+    public void shutdown(boolean sync) {
```

Comment:
Not sure it is a good idea to ask users to handle these two exceptions. They are shutting down tchannel, so I guess they won't care about these two exceptions at this stage.